projects/plugins/jetpack/_inc/client/components/apps-card/index.jsx

```diff
@@ -24,17 +24,62 @@ class AppsCard extends React.Component {
 		} );
 	};
 
-	dismissCard = () => {
-		this.props.dismissAppCard();
-		analytics.tracks.recordJetpackClick( {
-			target: 'apps-card',
-			button: 'dismiss',
-			page: this.props.location.pathname,
-		} );
+	getAppLinkSection = () => {
+		const device = detectMobileDevice();
+
+		switch ( device ) {
+			case 'ios':
+			case 'android':
+				return this.getAppCards( device );
+			case 'windows':
+			case 'unknown':
+				return this.getQrCode();
+			default:
+				return this.getQrCode();
+		}
 	};
 
+	getAppCards = device => (
+		<div className="jp-apps-card__apps-badges">
+			{ device === 'android' ? (
+				<AppsBadge
+					altText={ __( 'Google Play Store download badge.', 'jetpack' ) }
+					titleText={ __( 'Download the Jetpack Android mobile app.', 'jetpack' ) }
+					storeName="android"
+					storeLink="https://play.google.com/store/apps/details?id=com.jetpack.android&utm_source=jpdash&utm_medium=cta&utm_campaign=getappscard"
+					onBadgeClick={ this.trackDownloadClick }
+				/>
+			) : (
+				<AppsBadge
+					altText={ __( 'Apple App Store download badge.', 'jetpack' ) }
+					titleText={ __( 'Download the Jetpack iOS mobile app.', 'jetpack' ) }
+					storeName="ios"
+					storeLink="https://apps.apple.com/us/app/jetpack-website-builder/id1565481562?pt=299112ct=jpdash&mt=8"
+					onBadgeClick={ this.trackDownloadClick }
+				/>
+			) }
+		</div>
+	);
+
+	getQrCode = () => (
+		<div className="jp-apps-card__apps-qr-code">
+			<img src={ imagePath + 'get-apps-qr-code.svg' } alt="" />
+			<p className="jp-apps-card__caption">
+				{ createInterpolateElement(
+					__(
+						'Visit <a>jetpack.com/app</a> or scan this code to download the Jetpack mobile app.',
+						'jetpack'
+					),
+					{
+						a: <a className="jp-apps-card__link" href="https://jetpack.com/app" />,
```

Comment:
@SiobhyB  Looks like we should use the [Jetpack Redirects helper](https://github.com/Automattic/jetpack/tree/trunk/projects/js-packages/components/tools/jp-redirect) when adding external links instead of direct linking.

Reply:
@fluiddot, thanks! Addressed in https://github.com/Automattic/jetpack/pull/26668.